app/ui/window.js

```diff
@@ -98,28 +99,28 @@ module.exports = class Window {
         options
       );
 
-      const initSession = (opts, fn_) => {
-        fn_(uuid.v4(), new Session(opts));
-      };
-
-      initSession(sessionOpts, (uid, session) => {
+      window.initSession(sessionOpts, (uid, session) => {
```

Comment:
FWIW the functions aren't global, but they are members of the BrowserWindow class (this isn't the same window as the global window in browser JS).

It may be cleaner to not pollute the BrowserWindow object with these things and actually use the Window object (right now only the constructor is used).  If you like, I can do that in a follow-up PR.

Reply:
Yeah, sorry: we should not pollute BrowserWindow object.

We should add something in our API something like: `decorateSession(Session)`.
But I didn't study your plugin enough yet to find what is suitable or not.